src/server.cpp

```diff
@@ -648,13 +648,21 @@ void Server::AsyncRunStep(float dtime, bool initial_step)
 		MutexAutoLock lock(m_env_mutex);
 		// Figure out and report maximum lag to environment
 		float max_lag = m_env->getMaxLagEstimate();
-		max_lag *= 0.9998; // Decrease slowly (about half per 5 minutes)
-		if(dtime > max_lag){
-			if(dtime > 0.1 && dtime > max_lag * 2.0)
-				infostream<<"Server: Maximum lag peaked to "<<dtime
-						<<" s"<<std::endl;
-			max_lag = dtime;
+		// Decrease value gradually
```

Comment:
I do not generally like the fact the decay depends on the length the server step.
Why can't it decay by 1/2 in minute clock time (as measured by the server). I.e. we put dtime in the decay calculation.

I agree 5 minutes was way too long. One minute seems more appropriate. I could even 30s.

Reply:
I agree, I'll change the code to do it properly.